app/components/widgets/forms/duration-picker.js

```diff
@@ -0,0 +1,42 @@
+import $ from 'jquery';
+import Component from '@ember/component';
+import { merge } from 'lodash-es';
+import moment from 'moment';
+import { FORM_DURATION_FORMAT } from 'open-event-frontend/utils/dictionary/date-time';
+
+export default Component.extend({
+
+  classNames        : ['ui', 'calendar', 'time', 'picker', 'input', 'fluid', 'field'],
+  classNameBindings : ['icon:left', 'icon'],
+
+  icon: true,
+
+  today         : true,
+  rangePosition : 'none',
+  format        : FORM_DURATION_FORMAT,
+
+  options: {},
+
+  didInsertElement() {
+    this._super(...arguments);
+    const defaultOptions = {
+      type      : 'time',
+      formatter : {
+        time: date => {
+          if (!date) {return ''}
+          return moment(date).format(this.format);
+        }
+      }
+    };
+
+    $(this.element).calendar(merge(defaultOptions, this.options));
+  },
+  actions: {
+    onChange() {
+      if (this.onChange) {
+        this.sendAction('onChange', this.value);
+      }
+    }
+  }
+
+});
```

Comment:
so this file is totally similar to time-picker, only difference is` format        : FORM_DURATION_FORMAT,` if I am not wrong.

So try to pass format as prop in the timePicker for using it as durationPicker.

Reply:
Sounds good. Will do that, also are you getting any console errors due to this PR?

Reply:
I didn't check that, will see.